course_gather/models.py

```diff
@@ -88,18 +126,55 @@ class MTUCourse(models.Model):
     '''
     The MTU Course
     '''
-    mtu_course_id = models.CharField(
+    mtu_course_crn = models.CharField(
                             max_length=50,
-                            help_text='The id of the MTU course')
-    mtu_course_name = models.CharField(
-                            max_length=50,
-                            help_text='The Name of the MTU course')
-    mtu_subject = models.CharField(
+                            help_text='The CRN of the MTU course')
+    mtu_course_subject = models.CharField(
                         max_length=50,
                         help_text='The subject of the MTU course')
-    mtu_credits = models.FloatField(
+    mtu_course_id = models.CharField(
+                        max_length=10,
+                        help_text='The course ID of the MTU course')
+    mtu_course_section = models.CharField(
+                            max_length=3,
+                            help_text='The section of the MTU course')
+    mtu_course_campus = models.CharField(
+                            max_length=2,
+                            default="1",
+                            help_text='The campus code of the MTU course')
+    mtu_course_credits = models.FloatField(
                         blank=True,
                         help_text='The amount of credits the MTU course is')
+    mtu_course_name = models.CharField(
+                            max_length=50,
+                            help_text='The Name of the MTU course')
+    mtu_course_days = models.CharField(
+                            max_length=7,
+                            help_text='The days the class is scheduled for')
+    mtu_course_time = models.CharField(
+                            max_length=40,
+                            help_text='The hours the MTU course runs')
+    mtu_course_capacity = models.IntegerField(
+                            help_text='The capacity of the class')
+    mtu_course_capacity_enrolled = models.IntegerField(
+                            help_text='The amount of students currently \
+                            enrolled in the course')
+    mtu_course_capacity_remaining = models.IntegerField(
+                            help_text='The seats remaining in the course')
+    mtu_course_instructor = models.CharField(
+                            max_length=60,
+                            help_text='The last name of the professor(s) \
+                            teaching the course')
+    mtu_course_duration = models.CharField(
+                            max_length=30,
+                            help_text='The date range the class runs')
+    mtu_course_location = models.CharField(
+                            max_length=10,
+                            help_text='The MTU building location code the \
+                            class is located in')
+    mtu_course_fee = models.CharField(
+                            max_length=80,
```

Comment:
I'm pretty sure this value is always in dollars, so could we store this in cents instead of a string?

Reply:
It is almost always this field,
`Lab/Course Fee: $50.00 `

We could eventually try our hand at parsing this value in out, but that will have to happen in the scraper.

Reply:
Right, but since the scrapper isn't included in this PR might as well make it cents now. 🤷 

I guess it doesn't really matter that much.

Reply:
I just had an idea. We could preserve this data and make a real price column and have a tags system akin to the chips idea you originally posted. There exists other entries in the fee column such as:
```
On Line Technology Fee $114.00 and Lab/Course Fee: $20.00 
Lab/Course Fee:/ENG4910 $50.00
On Line Technology Fee $114.00 
Lab/CourseFee/ENG4900 $162.00
Lab/Course Fee: P1000 $15.00
```

Reply:
Going to do that and see how well it goes

Reply:
0335662